templates/docs/examples/base/code-inline-dark.html

```diff
@@ -0,0 +1,14 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Code - inline / dark{% endblock %}
+
+{% block standalone_css %}base{% endblock %}
+
+{% block content %}
+<div class="p-strip--dark is-dark">
```

Comment:
The dark strip uses only `p-strip--dark` class. It shouldn't have `is-dark` as well.